.pytool/Plugin/UncrustifyCheck/UncrustifyCheck.py

```diff
@@ -213,12 +247,14 @@ def _check_for_preexisting_formatted_files(self) -> None:
         Existence of such files is an unexpected condition. This might result
         from an error that occurred during a previous run or a premature exit from a debug scenario. In any case, the package should be clean before starting a new run.
         """
-        pre_existing_formatted_file_count = len(
-            [str(path.resolve()) for path in pathlib.Path(self._abs_package_path).rglob(f'*{UncrustifyCheck.FORMATTED_FILE_EXTENSION}')])
+        pre_existing_formatted_files = [str(path.resolve()) for path in pathlib.Path(self._abs_package_path).rglob(f'*{UncrustifyCheck.FORMATTED_FILE_EXTENSION}')]
+        pre_existing_formatted_file_count = len(pre_existing_formatted_files)
 
         if pre_existing_formatted_file_count > 0:
-            raise UncrustifyStalePluginFormattedFilesException(
-                f"{pre_existing_formatted_file_count} formatted files already exist. To prevent overwriting these files, please remove them before running this plugin.")
+            err_str  = f"{pre_existing_formatted_file_count} formatted files already exist. \n" \
+                       f"To prevent overwriting these files, please remove them before running this plugin. \n" \
+                       f"{'\n'.join(pre_existing_formatted_files) } \n"
+            raise UncrustifyStalePluginFormattedFilesException(err_str)
 

```

Comment:
I think this change is worth keeping regardless of dropping `_keep_formatted_file`. In the rare case files remain a list of those files is valuable.